Makefile

```diff
@@ -367,7 +367,7 @@ notices:
 	./scripts/append_license.sh frontend/src/assets/fonts/Source_Serif_Pro/Source-Serif-Pro.LICENSE
 	./scripts/append_license.sh frontend/src/assets/img/Material-Icons.LICENSE
 	./scripts/append_license.sh frontend/src/assets/img/Open-Iconic.LICENSE
-	./scripts/append_license.sh frontend/public/vendor/bokeh/bokeh-LICENSE.txt
+	./scripts/append_license.sh frontend/src/components/elements/BokehChart/vendor/bokeh-LICENSE.txt
```

Comment:
I'm not sure if this should move from public/vendor but if it should, I can move it back. I was thinking we don't have it in the index.html so we don't need it there anymore.

Reply:
it's not a huge deal (and others may disagree), but I do think we should keep all vendored files in that top-level `vendor/` directory. It's the pattern we're using already, and it also means that there's a simple one-stop shop for anyone trying to figure out what our vendored deps are.

(So: let's move the contents of `src/components/elements/BokehChart/vendor` into `public/vendor`)